skills/bear-hypermedia/SKILL.md

```
@@ -0,0 +1,146 @@
+
+    // 更新を実行
+    $updated = $this->resource->href('update', $edit, ['title' => 'New Title']);
+    $this->assertSame(200, $updated->code);
+}
+```
+
+## ユースケース例
+
+### 記事管理
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)